compiler/natives/src/reflect/reflect_test.go

```diff
@@ -187,3 +167,18 @@ func init() {
 	// TODO: This is a failure in 1.11, try to determine the cause and fix.
 	typeTests = append(typeTests[:31], typeTests[32:]...) // skip test case #31
 }
+
+func TestConvertNaNs(t *testing.T) {
+	// This test is exactly the same as the upstream, except it uses a "quiet NaN"
+	// value instead of "signalling NaN". JavaScript apepars to coerce all NaNs
```

Comment:
```suggestion
	// value instead of "signalling NaN". JavaScript appears to coerce all NaNs
```

Reply:
Quite the opposite, I appreciate them a lot! For some reason my brain is incapable of proper spell checking on myself, so your corrections are very welcome. Also, having typos checked in forever is a bit embarrassing, so correcting them at review time is preferable 🙂